### ISSUE TITLE
Depricated alias HomeAssistantType and function async_forward_entry_setup throw Warnings

### ISSUE THREAD

**Issue by amckenny** [jason0x43/hacs-hubitat#276]
**Describe the bug**

When loading Hubitat following a restart of Home Assistant, the following error is logged:
```
2024-07-07 14:04:25.896 WARNING (ImportExecutor_0) [homeassistant.helpers.typing] HomeAssistantType was used from hubitat, this is a deprecated alias which will be removed in HA Core 2025.5. Use homeassistant.core.HomeAssistant instead, please report it to the author of the 'hubitat' custom integration
2024-07-07 14:04:46.177 WARNING (MainThread) [homeassistant.helpers.frame] Detected that custom integration 'hubitat' calls async_forward_entry_setup for integration, hubitat with title: Hubitat (4d52a6b8) and entry_id: 95861519b0146f995cd7b851039bd67b, which is deprecated and will stop working in Home Assistant 2025.6, await async_forward_entry_setups instead at custom_components/hubitat/hub.py, line 317: hass.async_create_task(, please create a bug report at https://github.com/jason0x43/hacs-hubitat/issues
2024-07-07 14:04:46.204 WARNING (MainThread) [homeassistant.helpers.frame] Detected code that calls async_forward_entry_setup for integration hubitat with title: Hubitat (4d52a6b8) and entry_id: 95861519b0146f995cd7b851039bd67b, during setup without awaiting async_forward_entry_setup, which can cause the setup lock to be released before the setup is done. This will stop working in Home Assistant 2025.1. Please report this issue.
2024-07-07 14:04:46.206 WARNING (MainThread) [homeassistant.helpers.frame] Detected code that calls async_forward_entry_setup for integration hubitat with title: Hubitat (4d52a6b8) and entry_id: 95861519b0146f995cd7b851039bd67b, during setup without awaiting async_forward_entry_setup, which can cause the setup lock to be released before the setup is done. This will stop working in Home Assistant 2025.1. Please report this issue.
2024-07-07 14:04:46.215 WARNING (MainThread) [homeassistant.helpers.frame] Detected code that calls async_forward_entry_setup for integration hubitat with title: Hubitat (4d52a6b8) and entry_id: 95861519b0146f995cd7b851039bd67b, during setup without awaiting async_forward_entry_setup, which can cause the setup lock to be released before the setup is done. This will stop working in Home Assistant 2025.1. Please report this issue.
2024-07-07 14:04:46.216 WARNING (MainThread) [homeassistant.helpers.frame] Detected code that calls async_forward_entry_setup for integration hubitat with title: Hubitat (4d52a6b8) and entry_id: 95861519b0146f995cd7b851039bd67b, during setup without awaiting async_forward_entry_setup, which can cause the setup lock to be released before the setup is done. This will stop working in Home Assistant 2025.1. Please report this issue.
2024-07-07 14:04:46.230 WARNING (MainThread) [homeassistant.helpers.frame] Detected code that calls async_forward_entry_setup for integration hubitat with title: Hubitat (4d52a6b8) and entry_id: 95861519b0146f995cd7b851039bd67b, during setup without awaiting async_forward_entry_setup, which can cause the setup lock to be released before the setup is done. This will stop working in Home Assistant 2025.1. Please report this issue.
```

**To Reproduce**

1. Restart Home Assistant
2. Settings > System > Logs
3. Filter for "Hubitat"

**Environment**

 - Home Assistant version:
   - Core: 2024.7.1
   - Supervisor: 2024.06.2
   - Operating System: 12.4
   - Frontend: 20240705.0
 - Integration version:
   - HACS: 1.34.0
   - hacs-hubitat: 0.9.28

The offending lines appear to be: 
 - HomeAssistantType:
   - https://github.com/jason0x43/hacs-hubitat/blob/b16e85ef205adba8323aac29d22fa833d7d649c2/custom_components/hubitat/helpers.py#L4
   - https://github.com/jason0x43/hacs-hubitat/blob/b16e85ef205adba8323aac29d22fa833d7d649c2/custom_components/hubitat/helpers.py#L11
   - https://github.com/jason0x43/hacs-hubitat/blob/b16e85ef205adba8323aac29d22fa833d7d649c2/custom_components/hubitat/helpers.py#L22
   - https://github.com/jason0x43/hacs-hubitat/blob/b16e85ef205adba8323aac29d22fa833d7d649c2/custom_components/hubitat/helpers.py#L32
   - https://github.com/jason0x43/hacs-hubitat/blob/b16e85ef205adba8323aac29d22fa833d7d649c2/custom_components/hubitat/helpers.py#L41
 - async_forward_entry_setup:
   - https://github.com/jason0x43/hacs-hubitat/blob/b16e85ef205adba8323aac29d22fa833d7d649c2/custom_components/hubitat/hub.py#L318

**Comment by jason0x43** [comment]
Thanks for the report! This should be fixed in v0.9.29.

**Comment by markus99** [comment]
I'm on 0.9.31 and seeing this error:

Logger: homeassistant.helpers.typing
Source: helpers/deprecation.py:197
First occurred: 9:26:53 AM (1 occurrences)
Last logged: 9:26:53 AM

HomeAssistantType was used from hubitat, this is a deprecated alias which will be removed in HA Core 2025.5. Use homeassistant.core.HomeAssistant instead, please report it to the author of the 'hubitat' custom integration

I'm back on 2024.6.4, so unsure if my delay in upgrading to 2024.8.x is creating the issue, but guessing not.
